core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -297,7 +297,9 @@ private[spark] object JettyUtils extends Logging {
         (connector, connector.getLocalPort())
       }
       val httpConfig = new HttpConfiguration()
-      httpConfig.setRequestHeaderSize(conf.get(UI_REQUEST_HEADER_SIZE).toInt)
+      val requestHeaderSize = conf.get(UI_REQUEST_HEADER_SIZE).toInt
+      logDebug(s"Using requestHeaderSize: $requestHeaderSize")
+      httpConfig.setRequestHeaderSize(requestHeaderSize)
```

Comment:
Hi, @gaborgsomogyi . You need this in the older branches, too? Did I understand correct?

Reply:
You understand it correctly, it's needed in previous versions as well.